tests/check_framework/test_security.py

```diff
@@ -504,3 +504,17 @@ def test_failure_view_invalid_signature(self):
             csrf.check_csrf_failure_view(None),
             [Error(msg, id='security.E101')],
         )
+
+
+class CheckTypeOfAllowedHostsTest(SimpleTestCase):
+    @property
+    def func(self):
+        return base.check_type_of_allowed_hosts
```

Comment:
Please drop this property pattern and instead use plain function calls like in the above csrf failure view tests - we've been removing that pattern for checks tests